storage/postgresql/queue.go

```diff
@@ -0,0 +1,139 @@
+//go:build !batched_queue
+// +build !batched_queue
```

Comment:
You're the captain of this code, so you should strongly consider which implementation of this queue you want, and then delete the other one. If you think there's a good reason to keep both, then you have my blessing. But in reality, I suspect only one of these implementations will ever be used so best to make it a good one.

Reply:
Thanks.  I'm going with `queue_batching.go` then (although I'll rename it to `queue.go`).  Looks like this provides another 4x speed-up!!

Reply:
Addressed by https://github.com/google/trillian/pull/3644/commits/378779f016e2e829f1abd1011c9a85cd4f23e466.